### DISCUSSION TITLE
tuya wb3s device

### DISCUSSION THREAD

**Discussion by odedrefa** [arendst/Tasmota#16662] [Q&A] [answered]
Love Tasmota
but as far as I look all the new devices is wb3s chip based
I'm not an soldering expert so replace the chip is not an option for me
Tasmota is dead for me?

**Comment by s-hadinger** [comment, marked as answer]
Pretty much yes. Or switch to zigbee, it requires a gateway like Sonoff Zigbee Bridge Pro but appears more future-proof and no soldering.

**Reply by odedrefa** [reply]
So if I understand correctly you suggest to move to zigbee device and control them with Tasmota bridge?
In that case can I block the device to connect to any cloud?
Is there a guide to that method?
TIA

**Reply by sfromis** [reply]
With Zigbee via a coordinator with Tasmota, you do not need to block cloud access, as devices could not even try connecting.
https://tasmota.github.io/docs/Zigbee/

**Reply by sfromis** [reply]
There is also a modest selection of devices with factory-flashed:
https://templates.blakadder.com/preflashed.html

And at least Shelly devices are mostly very friendly to flashing via wires without soldering or even cracking it open. Most of the older ones can also work with OTA flashing.

**Comment by bitcloud** [comment]
Perhaps this could also be interesting for you?
https://community.home-assistant.io/t/detailed-guide-on-how-to-flash-the-new-tuya-beken-chips-with-openbk7231t/437276
It is not tasmota but another free and open firmware.

I have the same issue currently and I'm planning to give that one a spin when I'll find some time
